lib/collection/src/shards/local_shard/clock_map.rs

```diff
@@ -101,13 +101,21 @@ impl ClockMap {
         }
     }
 
-    /// Create a recovery point from the current clock map state
+    /// Create a recovery point based on the current clock map state, so that we can recover any
+    /// new operations with new clock values
+    ///
+    /// The recovery point will contain every clock that is in this clock map, but with a tick of
+    /// one higher. That is because we already have an operation for the current clock tick, but
+    /// would like to receive the operation with the next tick on recovery.
+    ///
+    /// In other words, the recovery point will contain the first clock tick values the clock map
+    /// has not seen yet.
     pub fn to_recovery_point(&self) -> RecoveryPoint {
         RecoveryPoint {
             clocks: self
                 .clocks
                 .iter()
-                .map(|(key, clock)| (*key, clock.current_tick))
```

Comment:
Can we test this new behavior? 
Off-by one changes are pretty subtle.

Reply:
Not at this time. But we will test it as part of our bigger testing efforts once all parts are in place.